substra/sdk/schemas.py

```diff
@@ -210,7 +207,7 @@ class ComputePlanSpec(_BaseComputePlanSpec):
 
     @staticmethod
     def compute_key() -> str:
```

Comment:
You can delete this, `ComputePlanSpec` inherits from `_BaseComputePlanSpec` that inherits from `_Spec` so the `compute_key` function is inherited.